### PR TITLE
LoadModelPass: provide public static function to read Model from llvm::Module

### PR DESCRIPTION
**Pull request by pfez**

### CODE REVIEW

File: include/revng/Model/LoadModelPass.h [hunk 1]
```diff
@@ -33,4 +35,6 @@ class LoadModelPass : public llvm::ImmutablePass {
     Modified = true;
     return TheBinary;
   }
+
+  static std::unique_ptr<model::Binary> getModel(const llvm::Module &M);
```
Comment (aleclearmind) [inline include/revng/Model/LoadModelPass.h:39]: Why all these `unique_ptr` business? If you're concerned about efficiency just use `std::move`.

Comment (pfez) [reply include/revng/Model/LoadModelPass.h]: Done!

**Comment by pfez** [comment]
I added two commits to do the following
- add a static method to `SerializeModulePass` to forcibly serialize the Model to a given `llvm::Module`
- use the newly added method to serialize an empty Model to the `llvm::Module`s lifted by `revng-lift`

**Comment by aleclearmind** [comment]
Looks good! Will merge soon!

**Comment by aleclearmind** [comment]
Merged in 10b90d6e155424d58a742c8b79c2042203624167.
Thanks!

**Pull request closed without merging**